--- ubermon.theme.orig	2004-12-21 21:54:05.268009488 +0100
+++ /ubermon1.0/ubermon.theme	2004-12-21 21:54:05.268009488 +0100
@@ -93,7 +93,7 @@
 text x=50 y=228 value="Up:" color=0,0,0 fontsize=9 font="Cure"
 text x=90 y=228 sensor=network device="eth0" format="%out kb/s" color=1,1,1 fontsize=9 font="Cure" interval=1000
 text x=50 y=238 value="IP:" color=0,0,0 fontsize=9 font="Cure"
-text x=90 y=238 sensor=program program="/home/life/programs/minimon-1.0/ubermon1.0/ipadd.pl"  color=1,1,1 fontsize=9 font="Cure"
+text x=90 y=238 sensor=program program="/usr/share/themes/superkaramba/ubermon/ipadd.pl"  color=1,1,1 fontsize=9 font="Cure"
 
 # Time Infomation
 text x=50 y=270 value="Uptime:" color=0,0,0 fontsize=9 font="Cure"
